triggers/commands/help.ts

```diff
@@ -6,8 +6,23 @@ import { hasPermission } from "../../Permissions";
 module.exports = <Command> {
     name: 'help',
     description: 'Send Help Message',
-    async execute(message) {
-
+    minArgs: 0,
+    async execute(message, args) {
+        
+        if (args.length > 0) {
+            commands.forEach(command => {
+                if (command.name == args[0]) {
+                    const embed = new MessageEmbed()
+                        .setColor("#AC2B37")
+                        .setTitle(command.name)
+                        .setDescription(command.description)
+                        .addField("Aliases", `${command.aliases ? command.aliases : 'None'}`)
+                        .addField("Required Permissions", `${command.requiredPerms == 'public' ? 'None' : 'Restricted'}`)
+                    message.channel.send(embed)
+                }
+            })
```

Comment:
I would attempt to use `getCommand` from `./util/commands` because it checks for aliases as well and there shouldn't be overlapping commands.